### ISSUE TITLE
Add Support for Extracted Comments

### ISSUE THREAD

**Issue by lf-jeremy** [rasmusbe/gulp-wp-pot#32]
gulp-wp-pot does not support extracted comments e.g. using the "translators:" (or other) keyword at the start of comments just prior to a translation function call, e.g.

_your-translated-file.php:_

``` php
<?php
/* translators: Limit to 32 characters or less */
$short_variable = __('Limited Length Phrase', 'sample-textdomain');
```

might produce a pot entry like so:

_sample-textdomain.pot: (excerpt)_

``` pot
#. translators: Limit to 32 characters or less
#: your-translated-file.php:3
msgid "Limited Length Phrase"
msgstr ""
```

**Comment by rasmusbe** [comment]
Thank you,

I think this would be good, but it requires a lot of work to make the script find it so I will have it in mind but I don't know when I can implement it. If someone can help implement it, it's always appreciated.

**Comment by lf-jeremy** [comment]
I think the added value is high enough that I will consider implementing it -- I'd like to make the keyword(s) configurable and let the parsing be tolerant cases like encountering opening/ending PHP tags, end of line comments the preceding line, inline comments, and either multi-line or multiple adjacent single line comments. It's enough complexity that perhaps forcing some limitations (e.g. only single-line comments) for an initial attempt would eliminate bugs.

**Comment by rasmusbe** [comment]
I'm working on a major rewrite right now that will fix this when it's released.

**Comment by lf-jeremy** [comment]
Excited to see this. I'll try to make use of the new support in my project once it makes it into a usable release.

**Comment by rasmusbe** [comment]
Fixed in 2.0. Defaults to `translators:` but can be changed with `commentKeyword` option
